### DISCUSSION TITLE
How to Pack/Unpack prost_types::Value and prost_types::Any?

### DISCUSSION THREAD

**Discussion by pionoor** [hyperium/tonic#1002] [Q&A]
In the generated rust code from the proto file, I have a struct which has a field:

```#[prost(message, optional, tag="3")]
    pub data: ::core::option::Option<::prost_types::Value>,
```

generated from a protobuff field:

   ``` google.protobuf.Value data = 3; ```

I can't seem to find a way to pack some var into `data` which is type `prost_types::Value`. 

There is another crate, [prost-wkt](https://github.com/fdeantoni/prost-wkt), which provide pack/unpack funcs for `Any`. But it won't generate grpc server from proto. Is there a way to tell `tonic_build` to use `prost-wkt-types` instead of `prost-types`? I tried to do:

```
    tonic_build::configure()
        .extern_path(".google.protobuf.Any", "::prost_wkt_types::Any")
        .extern_path(".google.protobuf.Value", "::prost_wkt_types::Value")
        .compile(&["proto/grpc_service.proto"], &["proto"])
        .unwrap();
```
But it the generated file didn't compile.

**Comment by LucioFranco** [comment]
Can you post the error you were getting when you made that config change?
